src/components/common/TmSessionExplore.vue

```diff
@@ -4,6 +4,32 @@
       <h2 class="session-title">
         Explore with any address
       </h2>
+
+      <div v-if="session.addresses.length > 0" class="session-list">
+        <div
+          v-for="account in session.addresses"
+          v-bind:key="account.address"
+          v-bind:title="account.address"
+          @click="exploreWith(account.address)"
+        >
+          <div class="tm-li-session">
+            <div class="tm-li-session-icon">
+              <i class="material-icons circle">{{
+                getAddressIcon(account.type)
+              }}</i>
+            </div>
+            <div class="tm-li-session-text">
+              <div class="tm-li-session-title">
+                <span>{{ shortenAddress(account.address) }}</span>
+              </div>
```

Comment:
this is awesome! can we add the "type" as a subfield so the user knows exactly what the icon means (should look like the networks page list items with a main title and subtitle below).

```
cosmos1xyzxyzxyzxyz
Lunie Browser Extension
```

```
cosmos1xyzxyzxyzxyz
Ledger Nano S
```

```
cosmos1xyzxyzxyzxyz
Explore Mode
```

Reply:
Done!

![image](https://user-images.githubusercontent.com/9256178/66002068-1daa1a00-e4a3-11e9-8d46-08627a4d45d0.png)